lib/gui/os/dropzone/directives/dropzone.js

```diff
@@ -42,16 +41,29 @@ module.exports = ($timeout) => {
     },
     link: (scope, element) => {
       const domElement = element[0];
+      const className = 'dropzone-hover';
```

Comment:
Maybe we can switch this to uppercase and make it clear in the name that this is a "hover" related class? E.g: `HOVER_CLASS_NAME`?